packages/ui/react-ui-list/src/components/List/List.stories.tsx

```diff
@@ -85,7 +85,7 @@ const SimpleStory = ({ items = [], ...props }: ListRootProps<TestItemType>) => {
 
 export default {
   // TODO(burdon): Storybook collides with react-ui/List.
-  title: 'react-ui-list/List',
+  title: 'react-ui-list-x/List',
```

Comment:
How does this collide? Isn’t react-ui’s List `'react-ui/List'`?

Reply:
it is, but they do still seem to collide. It seems to put them in the same group for some reason. I didn't want to spend time looking into it yet so that's why I added the x. Will try to figure it out before landing...

Reply:
Oh huh, weird…